options.ts

```diff
@@ -1120,8 +1120,9 @@ const options = mkOptions(OPTIONS, {
             showLabel: opt(true),
             lowBatteryThreshold: opt(20),
             lowBatteryNotification: opt(false),
-            lowBatteryNotificationText: opt(
-                'Your battery is running low ($POWER_LEVEL).\n\nPlease plug in your charger.',
+            lowBatteryNotificationText: opt('Warning: Low battery'),
+            lowBatteryNotificationTitle: opt(
+                'Your battery is running low ($POWER_LEVEL %).\n\nPlease plug in your charger.',
```

Comment:
I think the default title should be something like `Warning: Low Battery ($POWER_LEVEL%)` and the default text something like `Please plug in your PC`. Both of them being the same message looks a bit strange atm.

Reply:
![](https://i0.wp.com/www.omgubuntu.co.uk/wp-content/uploads/2016/07/battery-monitor-low-notificaiton.jpeg) takeing reference from some solutions like this

Reply:
looks good